packages/modals/src/elements/DrawerModal/DrawerModal.spec.tsx

```diff
@@ -176,4 +176,28 @@ describe('DrawerModal', () => {
 
     await waitFor(() => expect(queryByRole('dialog')).not.toBeInTheDocument());
   });
+
+  describe('focus management', () => {
+    it('correctly focuses on the Drawer when open', async () => {
+      const { getByText, getByRole } = render(<Example />);
+
+      await user.click(getByText('Open Drawer'));
+
+      expect(getByRole('dialog')).toStrictEqual(document.activeElement);
```

Comment:
```suggestion
      expect(getByRole('dialog')).toBe(document.activeElement);
```

suggested [nit] applies to changes below as well.

Reply:
Updated!